From 967978fe66d0c0788e62bf65ebe43bde77228125 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Fri, 3 Apr 2015 11:54:22 +0800
Subject: [PATCH 488/602] MA-6512 - [sabresd_6dq] on lp5.0, testCameraFeatures
 fail

Camera HAL report that flash is not available.
In kk4.4.3, will set flash mode to FLASH_MODE_OFF.
In lp5.0, KEY_FLASH_MODE will removed.
So remove feature aroid.hardware.camera.flash.

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 evk_6sl/required_hardware.xml       | 1 -
 sabreauto_6q/required_hardware.xml  | 1 -
 sabreauto_6sx/required_hardware.xml | 1 -
 sabresd_6dq/required_hardware.xml   | 1 -
 sabresd_6sx/required_hardware.xml   | 1 -
 sabresd_7d/required_hardware.xml    | 1 -
 6 files changed, 6 deletions(-)

diff --git a/evk_6sl/required_hardware.xml b/evk_6sl/required_hardware.xml
index adee65f..5cb012c 100644
--- a/evk_6sl/required_hardware.xml
+++ b/evk_6sl/required_hardware.xml
@@ -19,7 +19,6 @@
 -->
 <permissions>
     <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
     <feature name="android.hardware.microphone" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
diff --git a/sabreauto_6q/required_hardware.xml b/sabreauto_6q/required_hardware.xml
index 607483d..bba1b7a 100644
--- a/sabreauto_6q/required_hardware.xml
+++ b/sabreauto_6q/required_hardware.xml
@@ -18,7 +18,6 @@
 -->
 <permissions>
     <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
     <feature name="android.hardware.microphone" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
diff --git a/sabreauto_6sx/required_hardware.xml b/sabreauto_6sx/required_hardware.xml
index ddc076b..eb8ecf8 100644
--- a/sabreauto_6sx/required_hardware.xml
+++ b/sabreauto_6sx/required_hardware.xml
@@ -18,7 +18,6 @@
 -->
 <permissions>
    <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
     <feature name="android.hardware.microphone" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
diff --git a/sabresd_6dq/required_hardware.xml b/sabresd_6dq/required_hardware.xml
index 59ba7c6..0fd4fdf 100644
--- a/sabresd_6dq/required_hardware.xml
+++ b/sabresd_6dq/required_hardware.xml
@@ -19,5 +19,4 @@
 -->
 <permissions>
     <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
 </permissions>
diff --git a/sabresd_6sx/required_hardware.xml b/sabresd_6sx/required_hardware.xml
index 59ba7c6..0fd4fdf 100644
--- a/sabresd_6sx/required_hardware.xml
+++ b/sabresd_6sx/required_hardware.xml
@@ -19,5 +19,4 @@
 -->
 <permissions>
     <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
 </permissions>
diff --git a/sabresd_7d/required_hardware.xml b/sabresd_7d/required_hardware.xml
index adee65f..5cb012c 100644
--- a/sabresd_7d/required_hardware.xml
+++ b/sabresd_7d/required_hardware.xml
@@ -19,7 +19,6 @@
 -->
 <permissions>
     <feature name="android.hardware.camera.any" />
-    <feature name="android.hardware.camera.flash" />
     <feature name="android.hardware.microphone" />
     <feature name="android.software.home_screen" />
     <feature name="android.software.input_methods" />
-- 
1.8.0

